### DISCUSSION TITLE
[Feature] Ignore screenshots when displaying memories

### DISCUSSION THREAD

**Discussion by jordanlambrecht** [immich-app/immich#9772] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The dopamine rush doesn't hit nearly as hard when I get served a memory of a screenshot of a doordash order from 10 years ago.

Additionally, a weighted point system might be good for weighing the "luck" factor for what is served. For example:

On a curve of 100 assignable points,
- Photo of Person with second highest facial recognition image count: +66
- Photo containing self + another face: +50
- Photo of Self and no other face: +45
 - Face w/ same last name assigned: +66
- Object recognition of dog, cat: +25
- OR of mountain, ocean, beach, airplane, suitcase, glacier, crowd, concert, etc: +25
- Flower, plant, tree: +5
- OR of sunglasses: -5 (the eyes are the windows to the soul, after all)
- Toilet, trashcan, dumpster: -10
- No Person Detected: -33
- Text recognition of over 25 characters: -66
- No objects detected: -80
- Object recognition with anything that would throw a NSFW flag: -90
- Screenshot: -100

I'd be happy to work on a more comprehensive list

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Closing this in favour of #17162
